codex-rs/login/src/server.rs

```diff
@@ -50,6 +50,22 @@ pub struct LoginServer {
     pub shutdown_flag: Arc<AtomicBool>,
 }
 
+use std::mem::ManuallyDrop;
+
+impl Clone for LoginServer {
```

Comment:
This does not seem like something that should implement Clone. Can you wrap it in Arc instead?